bin/node-template/pallets/template/Cargo.toml

```diff
@@ -43,12 +49,12 @@ default-features = false
 version = "3.0.0"
 path = "../../../../primitives/runtime"
 
-
 [features]
 default = ['std']
 std = [
 	'codec/std',
 	'frame-support/std',
-	'frame-system/std'
+	'frame-system/std',
```

Comment:
I somehow did `cargo check --features runtime-benchmarks` at this path, and it failed with:
```
error[E0432]: unresolved import `sp_core::to_substrate_wasm_fn_return_value`
  --> primitives/api/src/lib.rs:65:9
   |
65 | pub use sp_core::to_substrate_wasm_fn_return_value;
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `to_substrate_wasm_fn_return_value` in the root

```
Either we add "frame-benchmarking/std" here or we just wait for feature on optional dependency